[BUG]org.opensearch.search.aggregations.bucket.TermsFixedDocCountErrorIT.testSliceLevelDocCountErrorMultiShard {p0={"search.concurrent_segment_search.enabled":"true"}} flaky

### Describe the bug

```
java.lang.AssertionError: expected:<2> but was:<3>
```

### Related component

Storage:Durability

### To Reproduce

```
REPRODUCE WITH: ./gradlew ':server:internalClusterTest' --tests "org.opensearch.search.aggregations.bucket.TermsFixedDocCountErrorIT" -Dtests.method="testSliceLevelDocCountErrorMultiShard {p0={"search.concurrent_segment_search.enabled":"true"}}" -Dtests.seed=41B9C6A508DBF95D -Dtests.security.manager=true -Dtests.jvm.argline="-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m" -Dtests.locale=es-MX -Dtests.timezone=Australia/North -Druntime.java=21
NOTE: leaving temporary files on disk at: /var/jenkins/workspace/gradle-check/search/server/build/testrun/internalClusterTest/temp/org.opensearch.search.aggregations.bucket.TermsFixedDocCountErrorIT_41B9C6A508DBF95D-001
NOTE: test params are: codec=Asserting(Lucene99): {s_value=Lucene99, index_uuid=Lucene99, _id=Lucene99, type=PostingsFormat(name=Asserting)}, docValues:{s_value=DocValuesFormat(name=Lucene90), _seq_no=DocValuesFormat(name=Asserting), _primary_term=DocValuesFormat(name=Lucene90), _version=DocValuesFormat(name=Lucene90)}, maxPointsInLeafNode=1593, maxMBSortInHeap=6.303042065496907, sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=es-MX, timezone=Australia/North
NOTE: Linux 5.15.0-1048-aws amd64/Eclipse Adoptium 21.0.1 (64-bit)/cpus=32,threads=1,free=460063632,total=933232640
NOTE: All tests run in this JVM: [RejectionActionIT, CloneIndexIT, GetIndexIT, ActiveShardsObserverIT, MinimumClusterManagerNodesIT, VotingConfigurationIT, ClusterSettingsIT, AliasedIndexDocumentActionsIT, HiddenIndexIT, MultiFieldsIntegrationIT, IndexingMemoryControllerIT, IndexPrimaryRelocationIT, UpdateNumberOfReplicasIT, SimpleNodesInfoIT, RemoteIndexRecoveryIT, RemoteStoreMultipartFileCorruptionIT, AggregationsIntegrationIT, GlobalIT, TermsFixedDocCountErrorIT]
```

### Expected behavior

Test to pass.

### Additional Details

https://build.ci.opensearch.org/job/gradle-check/32347/testReport/junit/org.opensearch.search.aggregations.bucket/TermsFixedDocCountErrorIT/testSliceLevelDocCountErrorMultiShard__p0___search_concurrent_segment_search_enabled___true___/

--- a/server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/TermsFixedDocCountErrorIT.java
+++ b/server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/TermsFixedDocCountErrorIT.java
@@ -23,6 +23,7 @@ import java.util.Arrays;
 import java.util.Collection;
 
 import static org.opensearch.common.xcontent.XContentFactory.jsonBuilder;
+import static org.opensearch.index.IndexSettings.MINIMUM_REFRESH_INTERVAL;
 import static org.opensearch.search.SearchService.CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING;
 import static org.opensearch.search.aggregations.AggregationBuilders.terms;
 import static org.opensearch.test.OpenSearchIntegTestCase.Scope.TEST;
@@ -71,7 +72,10 @@ public class TermsFixedDocCountErrorIT extends ParameterizedStaticSettingsOpenSe
         assertAcked(
             prepareCreate("idx_mshard_1").setMapping(STRING_FIELD_NAME, "type=keyword")
                 .setSettings(
-                    Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                    Settings.builder()
+                        .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                        .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        .put("index.refresh_interval", MINIMUM_REFRESH_INTERVAL)
                 )
         );
         client().prepareIndex("idx_mshard_1").setSource(jsonBuilder().startObject().field(STRING_FIELD_NAME, "A").endObject()).get();
@@ -89,7 +93,10 @@ public class TermsFixedDocCountErrorIT extends ParameterizedStaticSettingsOpenSe
         assertAcked(
             prepareCreate("idx_mshard_2").setMapping(STRING_FIELD_NAME, "type=keyword")
                 .setSettings(
-                    Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                    Settings.builder()
+                        .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                        .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        .put("index.refresh_interval", MINIMUM_REFRESH_INTERVAL)
                 )
         );
         client().prepareIndex("idx_mshard_2").setSource(jsonBuilder().startObject().field(STRING_FIELD_NAME, "A").endObject()).get();
@@ -127,7 +134,10 @@ public class TermsFixedDocCountErrorIT extends ParameterizedStaticSettingsOpenSe
         assertAcked(
             prepareCreate("idx_shard_error").setMapping(STRING_FIELD_NAME, "type=keyword")
                 .setSettings(
-                    Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                    Settings.builder()
+                        .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                        .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        .put("index.refresh_interval", MINIMUM_REFRESH_INTERVAL)
                 )
         );
         client().prepareIndex("idx_shard_error").setSource(jsonBuilder().startObject().field(STRING_FIELD_NAME, "A").endObject()).get();
@@ -170,7 +180,10 @@ public class TermsFixedDocCountErrorIT extends ParameterizedStaticSettingsOpenSe
         assertAcked(
             prepareCreate("idx_slice_error").setMapping(STRING_FIELD_NAME, "type=keyword")
                 .setSettings(
-                    Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                    Settings.builder()
+                        .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                        .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        .put("index.refresh_interval", MINIMUM_REFRESH_INTERVAL)
                 )
         );
         client().prepareIndex("idx_slice_error").setSource(jsonBuilder().startObject().field(STRING_FIELD_NAME, "A").endObject()).get();
@@ -248,7 +261,10 @@ public class TermsFixedDocCountErrorIT extends ParameterizedStaticSettingsOpenSe
         assertAcked(
             prepareCreate("idx_mshard_1").setMapping(STRING_FIELD_NAME, "type=keyword")
                 .setSettings(
-                    Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                    Settings.builder()
+                        .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                        .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        .put("index.refresh_interval", MINIMUM_REFRESH_INTERVAL)
                 )
         );
         client().prepareIndex("idx_mshard_1").setSource(jsonBuilder().startObject().field(STRING_FIELD_NAME, "A").endObject()).get();
@@ -288,7 +304,10 @@ public class TermsFixedDocCountErrorIT extends ParameterizedStaticSettingsOpenSe
         assertAcked(
             prepareCreate("idx_mshard_2").setMapping(STRING_FIELD_NAME, "type=keyword")
                 .setSettings(
-                    Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                    Settings.builder()
+                        .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                        .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        .put("index.refresh_interval", MINIMUM_REFRESH_INTERVAL)
                 )
         );
         client().prepareIndex("idx_mshard_2").setSource(jsonBuilder().startObject().field(STRING_FIELD_NAME, "A").endObject()).get();

